src/Analytics/Schema/Values.ts

```diff
@@ -50,6 +52,16 @@ export enum ActionType {
    * continuous hover mode.
    */
   Hover = "Hover",
+
+  /**
+   * A UI element was rendered in the viewport
+   */
+  Impression = "Impression",
+
+  /**
+   * Triggers a pageview in force, skips segment
+   */
+  ClickedReadMore = "Clicked read more",
```

Comment:
@eessex Why did you need to create a new `ActionType` here instead of using `Click`?

Reply:
This string maps to an [override in Force here](https://github.com/artsy/force/blob/dabc83fef5a266c14f6df9a4163023ba307c40d3/src/desktop/assets/analytics.coffee#L16) -- we will look at implementing segment's pageview in Reaction in the future, but for now this keeps the pageview firing.

Reply:
I see, thanks for the context